frontend/src/components/create-contract/ICreateContractForm.ts

```diff
@@ -8,6 +8,6 @@ export interface ICreateContractForm {
   labels: IListBoxItem[];
   identity: IListBoxItem[];
   acceptanceTests: IListBoxItem[];
-  deposit: string;
+  requiredDeposit: string;
```

Comment:
Why are we making the distinction between `requiredDeposit` and `deposit` now?

Reply:
In the smart contract, there is a job attribute called deposit that is the actual amount sent by the engineer.  In an attempt to avoid confusion, I renamed this requiredDeposit which is the minimum deposit required.

In the next PR, requiredDeposit is no longer a metadata field and becomes an attribute in the smart contract data.